### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by TheREK3R**
# Description

- Formally adopt cookbook into sous-chefs
- Add/fix tests

## Issues Resolved

Adds inspec tests. Removes old amazonlinux 1 platform

## Check List

- [X] All tests pass. See TESTING.md for details.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-description 'Installs and configures the Amazon linux yum repository'
-version '3.1.0'
+name              'yum-amazon'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
+license           'Apache-2.0'
+description       'Installs and configures the Amazon linux yum repository'
+version           '3.1.0'
+source_url        'https://github.com/sous-chefs/yum-centos'
+issues_url        'https://github.com/sous-chefs/yum-centos/issues'
```
Comment (ramereth) [inline metadata.rb:8]:
```suggestion
source_url        'https://github.com/sous-chefs/yum-amazon'
issues_url        'https://github.com/sous-chefs/yum-amazon/issues'
```

**Review by ramereth — CHANGES_REQUESTED** [review]
It looks like you removed support for version 2 and below. You should mention that in the ChangeLog and we should do a major bump when we release this.

File: metadata.rb (hunk 1 above)
Comment (TheREK3R) [reply metadata.rb:8]: oops

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [4.0.0](https://github.com/sous-chefs/yum-amazon/releases/tag/4.0.0)

**Pull request merged**